CONTRIBUTING.md

```diff
@@ -1,5 +1,29 @@
 # Contributing To Rez
 
+## Development Environment
+
+### Prerequisites 
+
+On Windows, make sure to set your PowerShell execution policy as shown [here](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.security/set-executionpolicy?view=powershell-5.1),
+otherwise the PowerShell tests will fail.
+
+### Setting Up
+
+To begin development on Rez you'll first need to set up your development environment. There are many 
+ways you can do it, but this is the recommended approach.
+
+1. Fork the repo and clone it.
+2. Create a new Git branch and check it out.
+3. Create a virtual environment in the same directory as the repo by running `python -m venv venv`.
+4. Pip install your local Rez code by running `pip install .`
```

Comment:
The virtual environment needs to be activated. It should also be noted of what's the different between this and installing using the `install.py` script.

In fact, I would encourage users to use the install script first, and then only mention `pip install .` later. The reason being that not all tests will be run if you `pip install .`.